### PR TITLE
Include userLink and data-user-id

### PR DESCRIPTION
**Pull request by Cyperghost**
Include the userLink and data-user-id by the popups of a user.

### CODE REVIEW

File: com.woltlab.wcf/templates/userInformationHeadline.tpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="containerHeadline">
-	<h3><a href="{link controller='User' object=$user}{/link}" class="username">{$user->username}</a>{if $user->banned} <span class="icon icon16 fa-lock jsTooltip jsUserBanned" title="{lang}wcf.user.banned{/lang}"></span>{/if}
-		{if MODULE_USER_RANK}
+	<h3><a href="{link controller='User' object=$user}{/link}" class="username userLink" data-user-id="{$user->userID}">{$user->username}</a>{if $user->banned} <span class="icon icon16 fa-lock jsTooltip jsUserBanned" title="{lang}wcf.user.banned{/lang}"></span>{/if}
+        {if MODULE_USER_RANK}
```
Comment (joshuaruesweg) [inline com.woltlab.wcf/templates/userInformationHeadline.tpl]: code style (whitespace instead of tabs)

**Comment by dtdesign** [comment]
@Cyperghost The userLink class is used for the user pop-over to identify trigger elements, but since popovers aren't allowed to show pop-overs within themselves, this change would have no effect.

Am I missing something?

**Comment by Cyperghost** [comment]
@dtdesign 
I need this, to mark the username as online, for my plugin.
But, i can use any other classes ;)

**Comment by dtdesign** [comment]
I did not ask you to change the class name, I was just asking _why_ you needed it, in case you missed my point. Please revert your last change.

**Comment by Cyperghost** [comment]
@dtdesign 
Sorry.

**Comment by dtdesign** [comment]
You've messed this up entirely, I've just made the change myself and referenced the PR instead ;)

**Pull request closed without merging**